ios/FluentUI/Tooltip/Tooltip.swift

```diff
@@ -133,6 +101,34 @@ open class Tooltip: NSObject {
         NotificationCenter.default.addObserver(self, selector: #selector(orientationChanged), name: UIDevice.orientationDidChangeNotification, object: nil)
     }
 
+    /// Displays a tooltip based on the current settings, pointing to the supplied anchorView.
+    /// If another tooltip view is already showing, it will be dismissed and the new tooltip will be shown.
+    ///
+    /// - Parameters:
+    ///   - message: The text to be displayed on the new tooltip view.
+    ///   - anchorView: The view to point to with the new tooltip's arrow.
+    ///   - preferredArrowDirection: The preferrred direction for the tooltip's arrow. Only the arrow's axis is guaranteed; the direction may be changed based on available space between the anchorView and the screen's margins. Defaults to down.
+    ///   - offset: An offset from the tooltip's default position.
+    ///   - screenMargins: The margins from the window's safe area insets used for laying out the tooltip. Defaults to 16.0 pts on all sides.
+    ///   - dismissMode: The mode of tooltip dismissal. Defaults to tapping anywhere.
+    ///   - onTap: An optional closure used to do work after the user taps
+    @available(*, deprecated, renamed: "show(with:title:for:preferredArrowDirection:offset:dismissOn:onTap:)")
```

Comment:
only the screenmargin is really deprecated. the rest we can still support as is.

Reply:
removing the deprecation for the method turns the usage of the deprecated screenmargins into a compiler error (not just a warning)